File: v2.x/Tutorials/Swift3/Tutorial02_CreatingASciChartSurface/HavingFunWithSciChart/ViewController.swift
//
//  ViewController.swift
//  HavingFunWithSciChart
//
//  Created by Yaroslav Pelyukh on 31/01/2017.
//  Copyright © 2017 SciChart Ltd. All rights reserved.
//

import UIKit
import SciChart

class ViewController: UIViewController {
    
    var chartView: SCIChartSurfaceView?
    var chartSurface: SCIChartSurface?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        chartView = SCIChartSurfaceView(frame: self.view.bounds)
        chartView?.translatesAutoresizingMaskIntoConstraints = true
        
        if let chartSurfaceView = chartView {
            self.view.addSubview(chartSurfaceView)
            
            chartSurface = SCIChartSurface(view: chartSurfaceView)
            
            chartSurface?.xAxes.add(SCINumericAxis())
            chartSurface?.yAxes.add(SCINumericAxis())
            
            chartSurface?.invalidateElement()
        }
    }
}

File: v2.x/Tutorials/Xamarin.iOS.C#/Tutorial02_CreatingSciChartSurface/CreatingSciChartSurface/CreatingSciChartSurface/ViewController.cs
using System;
using UIKit;
using SciChart.iOS.Charting;

namespace CreatingSciChartSurface
{
    public partial class ViewController : UIViewController
    {
        private SCIChartSurface _surface;

        public ViewController(IntPtr handle) : base(handle)
        {
        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
            // Perform any additional setup after loading the view, typically from a nib.
            _surface = new SCIChartSurface();
            _surface.TranslatesAutoresizingMaskIntoConstraints = true;
            _surface.Frame = this.View.Bounds;

            this.View.AddSubview(_surface);

            _surface.XAxes.Add(new SCINumericAxis());
            _surface.YAxes.Add(new SCINumericAxis());
        }
        
        public override void DidReceiveMemoryWarning()
        {
            base.DidReceiveMemoryWarning();
            // Release any cached data, images, etc that aren't in use.
        }
    }

}